Backend/app.py

```diff
@@ -157,7 +157,17 @@ def class_list():
 def joinClass():
   status = 200
   dynamoDBInstance = getDatabaseClient()
-  res = user_ctrl.joinClass(request.json, dynamoDBInstance)
```

Comment:
You should update user_ctrl to use the new logic here otherwise "join class" has two different meanings.